pkg/controller/hawtio/hawtio_controller.go

```diff
@@ -72,6 +76,7 @@ const (
 // Go build-time variables
 var ImageRepository string
 var LegacyServingCertificateMountVersion string
+var configVersion string
```

Comment:
Has `configVersion` to be declared as a package variable. It seems it can be a method variable. Besides, it is not used as a Go build-time variable.

Reply:
@astefanutti yup and right it is not a Go build-time variable. update the comments. the reason to make it global scope because deployment.Annotations[configVersionAnnotation value was setting empty at the local method variable scope.

Reply:
@astefanutti retrieve `configVersion` value before reconcile, https://github.com/redhatHameed/hawtio-operator/blob/f3e1b01b2333842ccf8ec830e4d7c2221d95aa4b/pkg/controller/hawtio/hawtio_controller.go#L354-L370 
LMK if that looks good.